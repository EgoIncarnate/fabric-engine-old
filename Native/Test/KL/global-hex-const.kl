const Size foo = 0x1234;
function entry() {
  report foo;
}
