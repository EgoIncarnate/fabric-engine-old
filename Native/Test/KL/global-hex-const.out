GlobalList
  GlobalConstDecl
    ConstSizeDecl( 'foo', Size, 4660 )
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        Report
          Var('foo')
4660
